README.md

```diff
@@ -56,12 +56,23 @@ Listed below are the exit codes and what they mean:
 - `7` - Invalid checksum of archive.
 - `8` - Cannot validate archive checksum.
 
+
+# Partial quicksync
```

Comment:
Btw, how about naming it _incremental_?

Reply:
No big preference. I used "partial" because the existing one is "full".